signup/tool/src/java/org/sakaiproject/signup/tool/jsf/SignupMeetingsBean.java

```diff
@@ -591,8 +591,14 @@ private void markingRecurMeetings(List<SignupMeetingWrapper> smList) {
 	 *         current user.
 	 */
 	public boolean isAllowedToDelete() {
-		if (getSignupMeetings() == null)
+		if (sakaiFacade.isAllowedSite(sakaiFacade.getCurrentUserId(), SakaiFacade.SIGNUP_DELETE_SITE, sakaiFacade.getCurrentLocationId())) {

```

Comment:
If an instructor is a member of 200 sites, doesn't this shortcut avoid having to fully load 200 sites?